src/openzaak/components/zaken/openapi.yaml

```diff
@@ -2023,6 +2023,27 @@ paths:
       operationId: rol_delete
       summary: Verwijder een ROL van een ZAAK.
       description: Verwijder een ROL van een ZAAK.
+      parameters:
```

Comment:
@joeribekker these are missing in the RI, they'll show up with the final check for #153 - it's a bug in the 1.0.x standard to be fixed.